crypto/schemes.go

```diff
@@ -248,6 +250,44 @@ func NewPedersenBLSUnchainedG1() (cs *Scheme) {
 	}
 }
 
+// BN254UnchainedOnG1SchemeID is the scheme id used to set unchained randomness on beacons with signatures on G1,
+// on the BN254 curve.
+const BN254UnchainedOnG1SchemeID = "bls-bn254-unchained-on-g1"
+
+// NewPedersenBLSBN254UnchainedOnG1Scheme instantiates a scheme of type "bls-bn254-unchained-on-g1" which is also
+// unchained, only hashing the round number as the message being signed in beacons. This scheme is configured to
+// be optimally compatible with the EVM.
+func NewPedersenBLSBN254UnchainedOnG1Scheme() (cs *Scheme) {
+	var Pairing = bn254.NewSuite()
+	Pairing.SetDomainG1([]byte("BLS_SIG_BN254G1_XMD:KECCAK-256_SSWU_RO_NUL_"))
+
```

Comment:
Is there really no need to set the G2 domain too?

Reply:
Since the DST is only used for hashing messages, the G2 DST would never be used for this particular scheme (and the BN254 code in Kyber only supports mapping to G1). Should I add it anyway?

Reply:
sg